### PR TITLE
65 feature 21 give option to go back to menu when game is won

### PR DESCRIPTION
**Pull request by SMateen21**
Created a pop up that comes when the game has been won, it shows who has won and a button to go back to the main menu.

### CODE REVIEW

File: src/main/java/ui/windows/game/GameDisplayData.java [hunk 1]
```diff
@@ -6,17 +6,23 @@ public class GameDisplayData {
...
         this.cards = cards;
         this.currentCard = currentCard;
+        this.hasWinner = hasWinner;
     }
 
     public String getCurrentPlayer() {
         return currentPlayer;
     }
 
+    public boolean getHasWinner() {
```
Comment (meiiph) [inline src/main/java/ui/windows/game/GameDisplayData.java:22]: The other methods should have documentation from #76, but please add some to this method. :)

File: src/main/java/ui/windows/game/GameDisplay.java [hunk 2]
```diff
@@ -70,6 +80,23 @@ public void updateView(GameDisplayData data) {
         currentCard.setVisible(true);
     }
 
+    private void winScreen() {
```
Comment (meiiph) [inline src/main/java/ui/windows/game/GameDisplay.java:83]: Please add javadocs to this method :)

File: src/main/java/ui/windows/game/GameDisplay.java [hunk 3]
```diff
@@ -70,6 +80,23 @@ public void updateView(GameDisplayData data) {
...
+        int num = JOptionPane.showOptionDialog(this, winner + " has won!",
+                "Game is over", JOptionPane.YES_OPTION, JOptionPane.INFORMATION_MESSAGE,
+                null, buttons, 0);
+        if(num == JOptionPane.YES_OPTION) {
+            goToMenu.doClick();
+        }
+
+    }
+
+    public void setNavigator(PaneDelegator navigator) {
```
Comment (meiiph) [inline src/main/java/ui/windows/game/GameDisplay.java:96]: Javadocs please :)

**Review by meiiph — DISMISSED** [review]
My javadocs comments have been addressed 👍

**Review by nyan-ko — APPROVED** [review]

**Review by nyan-ko — APPROVED** [review]

**Review by meiiph — APPROVED** [review]

**Review by lilyxmeng — APPROVED** [review]

**Pull request merged**